1-js/13-modules/02-import-export/article.md

```diff
@@ -90,159 +89,159 @@ say.sayHi('John');
+Agora `hi` e `bye` são nomes oficias para quem vê de fora, para serem usadas nos imports:
 
 ```js
 // 📁 main.js
 import * as say from './say.js';
 
-say.hi('John'); // Hello, John!
-say.bye('John'); // Bye, John!
+say.*!*hi*/!*('John'); // Olá, John!
+say.*!*bye*/!*('John'); // Tchau, John!
 ```
 
-## export default
-
-So far, we've seen how to import/export multiple things, optionally "as" other names.
+## Export default
 
-In practice, modules contain either:
-- A library, pack of functions, like `lib.js`.
-- Or an entity, like `class User` is described in `user.js`, the whole module has only this class.
+Na prática, exitem principalmente dois tipos de módulos.
 
-Mostly, the second approach is preferred, so that every "thing" resides in its own module.
+1. Módulos que contém uma biblioteca, pacote de funções, como `say.js` acima.
+2. Módulos que declaram uma única entidade, por exemplo um módulo `user.js` exportando apenas `class User`.
 
-Naturally, that requires a lot of files, as everything wants its own module, but that's not a problem at all. Actually, code navigation becomes easier, if files are well-named and structured into folders.
+Na maioria dos casos, a segunda abordagem é preferida, para que cada "coisa" tenha seu próprio módulo.
 
-Modules provide special `export default` syntax to make "one thing per module" way look better.
+Naturalmente isso exige muitos arquivos porque tudo precisa do seu próprio módulo, mas isso não é problema. Na verdade a navegação do código se torna mais fácil se os arquivos forem bem nomeados e estruturados em pastas.
 
-It requires following `export` and `import` statements:
+Módulos fornecem a sintaxe especial `export default` ("a exportação default") para melhorar a aparência de "uma coisa por módulo".
 
-1. Put `export default` before the "main export" of the module.
-2. Call `import` without curly braces.
-
-For instance, here `user.js` exports `class User`:
+Coloque `export default` antes da entidade a ser exportada:
 
 ```js
 // 📁 user.js
-export *!*default*/!* class User { // just add "default"
+export *!*default*/!* class User { // apenas adicione "default"
   constructor(name) {
     this.name = name;
   }
 }
 ```
 
-...And `main.js` imports it:
+Pode existir apenas um `export default` por arquivo.
+
+...E então importamos sem as chaves:
 
 ```js
 // 📁 main.js
-import *!*User*/!* from './user.js'; // not {User}, just User
+import *!*User*/!* from './user.js'; // não {User}, apenas User
 
 new User('John');
 ```
 
-Imports without curly braces look nicer. A common mistake when starting to use modules is to forget curly braces at all. So, remember, `import` needs curly braces for named imports and doesn't need them for the default one.
+Imports sem chaves parecem mais agradáveis. Um erro comum quando se começa a usar módulos é esquecer das chaves. Então, lembre-se, `import` precisa de chaves para import nomeados e não precisa para os imports default.
```

Comment:
I believe it would be better to use "imports nomeados (named imports)". What do you think?

Reply:
I agree with that.